tools/pkglint/package.json

```diff
@@ -34,7 +34,6 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/colors": "^1.2.1",
```

Comment:
why?

Reply:
`colors` has shipped with built-in type definitions for a while now... `@types/colors` is only an empty shim now.